src/Spout/Reader/ODS/RowIterator.php

```diff
@@ -161,7 +196,9 @@ public function next()
             throw new IOException("The sheet's data cannot be read. [{$exception->getMessage()}]");
         }
 
-        $this->rowDataBuffer = $rowData;
+        for ($i = 0; $i < $numRowsRepeated; ++$i) {
```

Comment:
This will be a problem. If you have one row repeated 100,000 times, you will have memory problems as you'll end up storing 100,000 rows in memory.